packages/insomnia/src/ui/routes/error.tsx

```diff
@@ -46,13 +45,7 @@ export const ErrorRoute: FC = () => {
     <Container>
       <h1 style={{ color: 'var(--color-font)' }}>Application Error</h1>
       <p style={{ color: 'var(--color-font)' }}>
-        Failed to render. Please send the following error to{' '}
-        <Mailto
-          email="support@insomnia.rest"
-          subject="Error Report"
-          body={errorMessage}
-        />
-        .
+        Failed to render. Please report to Github Issues: https://github.com/Kong/insomnia/issues
```

Comment:
Nit: We could add a clickable link:
```suggestion
        Failed to render. Please report to Github Issues: <a href="https://github.com/Kong/insomnia/issues">https://github.com/Kong/insomnia/issues</a>
```